### PR TITLE
Added OWNERS files for Huskies

### PR DESCRIPTION
**Pull request by igor-karpukhin**
This PR relates to the Tide integration process when OWNERS file that will replace CODEOWNERS.
Updated components: monitoring, logging

### CODE REVIEW

File: resources/logging/OWNERS [hunk 1]
```diff
@@ -0,0 +1,7 @@
+approvers:
+ - hisarbalik
+ - clebs
+ - suleymanakbas91
+ - igor-karpukhin
```
Comment (suleymanakbas91) [inline resources/logging/OWNERS]: don't forget lilit :)

Comment (igor-karpukhin) [reply resources/logging/OWNERS]: She is in the list, one line below :)

File: resources/backup-init/OWNERS [hunk 2]
```diff
@@ -0,0 +1,9 @@
+approvers:
+ - hisarbalik
+ - clebs
+ - suleymanakbas91
+ - igor-karpukhin
+ - lilitgh
+ - a-thaler
+labels:
```
Comment (suleymanakbas91) [inline resources/backup-init/OWNERS]: wrong area

File: resources/backup/OWNERS [hunk 3]
```diff
@@ -0,0 +1,10 @@
+approvers:
+ - hisarbalik
+ - clebs
+ - suleymanakbas91
+ - igor-karpukhin
+ - lilitgh
+ - a-thaler
+labels:
+ - area/tracing
```
Comment (suleymanakbas91) [inline resources/backup/OWNERS]: wrong area

File: resources/backup-init/OWNERS (hunk 2 above)
Comment (igor-karpukhin) [reply resources/backup-init/OWNERS]: Fixed

File: resources/backup/OWNERS (hunk 3 above)
Comment (igor-karpukhin) [reply resources/backup/OWNERS]: Fixed

**Pull request merged**